extensions/bigtable/install_hbase_bigtable.sh

```diff
@@ -78,3 +78,55 @@ echo -e "YARN_OPTS=\"\${YARN_OPTS} -Dyarn.app.mapreduce.am.command-opts=\"${BIGT
 # the hbase classpath.
 echo -e "HBASE_OPTS=\"\${HBASE_OPTS} ${BIGTABLE_BOOT_OPTS}\"" >> "${HBASE_CONF_DIR}/hbase-env.sh"
 
+# Create bigtable wrapper shell scripts for spark-submit and spark-shell 
+if [ ! -z "${SPARK_INSTALL_DIR:-}" ]; then 
+    echo -e "#!/usr/bin/env bash" > "${HBASE_INSTALL_DIR}/bin/execute-with-bigtable"
+    cat << EOF >> "${HBASE_INSTALL_DIR}/bin/execute-with-bigtable"
+EXECUTE_OPT="\$1"; shift
+if [[ "\${EXECUTE_OPT}" != "spark-submit" && "\${EXECUTE_OPT}" != "spark-shell" ]]; then
+    echo "Please enter 'spark-submit' or 'spark-shell' as the first argument when calling execute-with-bigtable."
+    exit 1
+fi
+if [[ "\${EXECUTE_OPT}" = "spark-submit" ]] &&  ! which "spark-submit" ; then
+    echo "Cannot find spark-submit on the \\\$PATH."
+    exit 1
+fi
+if [[ "\${EXECUTE_OPT}" = "spark-shell" ]] && ! which "spark-shell" ; then
+    echo "Cannot find spark-shell on the \\\$PATH."
+    exit 1
+fi
+CONFIG_OPTS=()
+APPLICATION_ARGS=()
+EXTRA_CLASSPATH=""
+contains_jars=false
+contains_extra_jars=false
+
+while ((\$#)); do
+    case "\$1" in
+        --jars)
+	        contains_jars=true
+            CONFIG_OPTS+=("\$1"); shift
+            CONFIG_OPTS+=("\$1"); shift
```

Comment:
Yes. The first line is to get the "--jars", the second line is to get the jars that come after "--jars". Because Spark can have other --ARGUMENT, it needs the --jars to tell what the jars coming after --jars are for.